fighthealthinsurance/rest_views.py

```
@@ -550,6 +552,108 @@ def _sanitize(val: str, max_len: int) -> str:
+                                status_count += 1
+                                last_status_phase = parsed.get("phase")
+                    except (json.JSONDecodeError, TypeError):
+                        pass
+            if appeal_count == 0:
+                await log_zero_appeal_diagnostics(
+                    denial_id=denial_id,
+                    status_count=status_count,
+                    last_status_phase=last_status_phase,
+                    transport="rest",
+                )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)